### PR TITLE
KO: translate ko/r2/guide/effective_tf2

### PR DESCRIPTION
**Pull request by rickiepark**
Korean Translation of effective_tf2.md
I will update it in a while for proofreading.
Thanks.

### CODE REVIEW

**Comment by rickiepark** [comment]
I finished effective_tf2 translation. :)

File: site/ko/r2/guide/effective_tf2.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+# 이펙티브 텐서플로 2.0
+
+> 노트: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 [공식 영문 문서](https://github.com/tensorflow/docs/blob/master/site/en/r2/guide/effective_tf2.md)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 [tensorflow/docs](https://github.com/tensorflow/docs) 깃헙 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 지원하려면 [이 양식](https://bit.ly/tf-translate)을 작성하거나 [docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)로 메일을 보내주시기 바랍니다.
```
Comment (lamberta) [inline site/ko/r2/guide/effective_tf2.md]: This must remain as 'Note: '
It's a markup command to render the callout box on tensorflow.org

**Comment by rickiepark** [comment]
Thanks for comment.
I just push a commit to restore 'Note:'.
I will update the translation according to the changes in en folder.
Thank you. :)

File: site/ko/r2/guide/effective_tf2.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # 이펙티브 텐서플로 2.0
 
-> Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 [공식 영문 문서](https://github.com/tensorflow/docs/blob/master/site/en/r2/guide/effective_tf2.md)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 [tensorflow/docs](https://github.com/tensorflow/docs) 깃헙 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 지원하려면 [이 양식](https://bit.ly/tf-translate)을 작성하거나 [docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)로 메일을 보내주시기 바랍니다.
```
Comment (rickiepark) [inline site/ko/r2/guide/effective_tf2.md]: Thanks. It's my mistake. :(

**Comment by rickiepark** [comment]
Hi @lamberta , `en/r2/tutorial/beginner/tf2_overview.ipynb` link is broken in en/r2/guide/effective_tf2.md.
Just remove "For complete examples, see MNIST (basic example)" or link other notebook?

**Pull request merged**